syntax = "proto3";

package block;

service Block {
	rpc WriteBlock(BlockRequest) returns (BlockReply) {}
	rpc GetStat(StatRequest) returns (StatReply) {}
}

message BlockRequest {
	int32 size = 1;
	int32 start_block = 2;
	bytes data = 3;
}

message BlockReply {
}

message StatRequest {

}

message StatReply {
	bytes stat = 1;
}